content/en/docs/architecture/timeouts.md

```diff
@@ -17,6 +17,12 @@ Timeouts occur for a job when the overall job or some subset of the job exceeds
 
 Furthermore, instability in the OpenShift build farm clusters hosting these workloads may also trigger the deletion of `Pods` that implement a job. In these cases, the resulting workflow is identical, as `Pod` eviction honors the configured grace periods.
 
+{{< alert title="Note" color="info" >}}
+Part of the CI implementation involves one or more small coordinating processes executed inside of each test `Pod`. This is the case for Prow jobs which execute `ci-operator` or any other program, as well as for the individual steps in a `ci-operator` test.
```

Comment:
> one or more small coordinating processes

Could you link both the `entrypoint` and the `entrypoint-wrapper` such that it would be easier for the reader to wrap his head around this idea?

Reply:
This is exactly what led to create the next pull request (#327), which adds an entire "internals" page about the implementation details.

See the same note in https://deploy-preview-327--zealous-shirley-0f56b3.netlify.app/docs/architecture/timeouts#architecture.